code/modules/psionics/psion_powers/locator_powers.dm

```diff
@@ -16,7 +16,7 @@
 	if (isnull(target))
 		return
 
-	if(pay_power_cost(psi_point_cost))
+	if(pay_power_cost(psi_point_cost) && check_possibility()) //Doesn't use the normal check possibility flags since we want it to work on dead people
 		if(target.psi_blocking >= 10)
```

Comment:
this still needed? Why not put it into the check_possibility check?

Reply:
I could just write a bespoke "works on dead" line into the proc, I suppose, just didn't think it was particularly necessary for a single-use case.